.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+__tasks:__
+
+- [ ] test coverage
+- [ ] submit changes to the documentation
+- [ ] ... <add your own tasks>
```

Comment:
Can you only leave test coverage here? `<add your own tasks>` always has to be removed anyway, and we already have a Document PR line in the information table.